src/components/user-details/user-details.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { ProfilePicture } from '@/components/profile-picture/profile-picture';
+import './user-details.scss';
+
+interface UserDetailsProps {
+  username: string;
+  email?: string;
+  yearJoined: number;
```

Comment:
maybe this should be `dateJoined` and be a Date? we could display it using `getFormattedDate`
i feel like year is too little info and we're gonna parse out a Date from the response eventually anyway

Reply:
Yeah, that is a good idea. It's fixed now